python/pyspark/sql/tests/test_pandas_udf_grouped_agg.py

```diff
@@ -491,6 +491,28 @@ def max_udf(v):
             agg2 = self.spark.sql("select max_udf(id) from table")
             assert_frame_equal(agg1.toPandas(), agg2.toPandas())
 
+    def test_no_predicate_pushdown_through(self):
+        from pyspark.sql.functions import monotonically_increasing_id, explode_outer
```

Comment:
@viirya, I think we should better leave a comment like:

```
def test_case(self):
    # SPARK-12345: a short description of the test
    ...
```

per the new guide written recently (https://spark.apache.org/contributing.html)

Reply:
Sure. Thanks for reminding me.